pkgs/top-level/all-packages.nix

```diff
@@ -23533,6 +23533,8 @@ in
     opencv = python3Packages.opencv3;
   };
 
+  manim-community = python3Packages.callPackage ../applications/video/manim-community { };
```

Comment:
```suggestion
  manim-community = callPackage ../applications/video/manim-community { };
```

Reply:
okay, will do that

Reply:
You'll have to use `python3.pkgs.buildPythonApplication`.
If use use `python3Packages.callPackage` overriding specific Python dependencies will be harder.

Reply:
done